src/lsst/cmservice/web_app/app.py

```diff
@@ -71,11 +71,12 @@ async def get_campaigns(
 ) -> HTMLResponse:
     try:
         async with session.begin():
-            campaigns = await db.Campaign.get_rows(session)
-            campaigns_list = []
-            for campaign in campaigns:
-                campaign_details = await get_campaign_details(session, campaign)
-                campaigns_list.append(campaign_details)
+            # campaigns = await db.Campaign.get_rows(session)
+            # campaigns_list = []
+            # for campaign in campaigns:
+            #     campaign_details =
+            #     await get_campaign_details(session, campaign)
+            #     campaigns_list.append(campaign_details)
```

Comment:
Generally, Rubin DM style guide strongly discourages leaving commented-out sections of code.  If you are intending to return here directly right away, it can be okay...

Reply:
Removed. Thanks Fritz!